boefjes/boefjes/plugins/kat_burpsuite/normalize.py

```diff
@@ -16,15 +15,9 @@
 
 
 def run(input_ooi: dict, raw: bytes) -> Iterable[NormalizerOutput]:
-    parser = minidom.parse(raw)
-
-    # assume that input ooi is none or a HostnameHTTPURL
-    if input_ooi:
-        ooi = Reference.from_str(input_ooi["primary_key"])
-        network = Network(name=ooi.tokenized.netloc.network.name)
-    else:
-        network = Network(name="internet")
-        yield network
+    parser = minidom.parseString(raw.decode("UTF-8"))
+
+    network = Network(name="internet")
```

Comment:
Burpsuite seems to be one of those tools that would be used on the internal networks a lot.

Reply:
I can make support for either scan oois -> internet or hostnames -> hostname.network